test/el_cl_launcher_test.star

```diff
@@ -75,11 +75,13 @@ def test_launch_with_defaults(plan):
         da_server_context=da_server_context,
     )
 
-    el_service_name = "op-el-1-op-reth-op-node-"
+    el_service_name = "op-el-2151908-1-op-reth-op-node-"
```

Comment:
What's this special number?

Reply:
chainid I believe

Reply:
Btw these kinds of changes are breaking for us because our justfile had hardcoded names for port-finding:
```
  ROLLUP_NODE_RPC="$(kurtosis port print {{enclave}} op-cl-1-op-node-op-geth-op-kurtosis http)"
```

Would be nice if kurtosis service names/port command supported grep expressions but they don't afaiu. Could `kurtosis enclave inspect | grep` instead but that is super slow so very annoying.